client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/InvestigationTableFooter.tsx

```diff
@@ -76,22 +78,24 @@ const InvestigationTableFooter: React.FC<Props> = React.forwardRef((props: Props
     }, [checkedInvestigations])
 
     const shouldGroupActionDisabled: boolean = useMemo(() => {
-        return Boolean(checkedInvestigations.find((investigationToGroup: InvestigationTableRow) => investigationToGroup.groupId !== null))
+        const groupIds = checkedInvestigations.map((invetigationToGroup: InvestigationTableRow) => invetigationToGroup.groupId);
+        const trimedGroupidIds = Array.from(new Set(groupIds));
+        return trimedGroupidIds.length > 2 || (trimedGroupidIds.findIndex((groupId: string) => groupId === null) === -1) || checkedInvestigations.length < 2
```

Comment:
The number two is also used on another place. To make it clearer I would export it to a const and use it on both places

Reply:
In addition change include here as well